vars/Alpine.yml

```diff
@@ -17,3 +17,6 @@ alpine_glibc_pkg_names:
   - glibc-bin
   - glibc-dev
   - glibc-i18n
+
+alpine_update_alternatives_deps:
+  - dpkg
```

Comment:
Maybe this should be with the other `system_packages`? If there's no reason not to, then maybe this should just be moved into `system_packages` to keep things simple.